File: src/main/java/com/example/ranyoungonboarding/repository/AnswerRepository.java
package com.example.ranyoungonboarding.repository;

import com.example.ranyoungonboarding.domain.Answer;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface AnswerRepository extends JpaRepository<Answer, Long> {
    List<Answer> findByResponseId(Long responseId);
    List<Answer> findByQuestionId(Long questionId);
}

File: project/seonghun-onboarding/src/main/kotlin/com/chosseang/seonghunonboarding/repository/AnswerRepository.kt
package com.chosseang.seonghunonboarding.repository

import com.chosseang.seonghunonboarding.entity.Answer
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface AnswerRepository : JpaRepository<Answer, Long> {
}
